package.json

```diff
@@ -7,9 +7,9 @@
   ],
   "license": "MIT",
   "scripts": {
-    "build": "yarn run build:cjs && yarn run build:esm && yarn run build:types",
-    "build:cjs": "yarn run beemo babel --workspaces=*",
-    "build:esm": "yarn run beemo babel --workspaces=* --esm",
+    "build": "yarn run build:cjs && yarn run build:esm && yarn run build:types && yarn build:tokens",
+    "build:cjs": "yarn run beemo babel --workspaces='*/!(tokens)'",
```

Comment:
need to ignore the `tokens` package since that doesn't follow the standard babel build flow. glob patterns are cool.